### PR TITLE
New package: libskk

### PR DESCRIPTION
**Pull request by rail44**
SKK is system of Japanese input method.
This package is used by input method such as ibus-skk or fcitx-skk.

### CODE REVIEW

File: srcpkgs/libskk/template [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+maintainer="Satoshi Amemiya <amemiya@protonmail.com>"
+homepage="https://github.com/ueno/${pkgname}"
+license="GPL-3"
+hostmakedepends="pkg-config intltool gnome-common gobject-introspection vala gettext-devel"
+makedepends="libgee08-devel json-glib-devel"
+build_style=gnu-configure
+distfiles="https://github.com/ueno/${pkgname}/archive/${version}.tar.gz"
+checksum=5a68d100e006c3525e7f94579428c1cf015320f4f2e879e18c470dce3cba93c5
+
+configure_args="--prefix=/usr"
```
Comment (Vaelatern) [inline srcpkgs/libskk/template]: Line redundant.

Comment (rail44) [inline srcpkgs/libskk/template]: I see.

**Comment by rail44** [comment]
And, I cannot find reason what makes fail packaging on some of architecture.
(I'm sorry unfamiliar with xbps)

On my local machine, It is success with cross building.
Could you find and tell me that something found?

**Comment by rail44** [comment]
I've just found libgee08-devel is necessary also on host machine :sweat_smile:

File: srcpkgs/libskk/template [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Template file for 'lib-skk'
+pkgname=libskk
+version=1.0.2
+revision=1
+short_desc="Japanese SKK input method library"
+maintainer="Satoshi Amemiya <amemiya@protonmail.com>"
+homepage="https://github.com/ueno/${pkgname}"
+license="GPL-3"
+hostmakedepends="libgee08-devel pkg-config intltool gnome-common gobject-introspection vala gettext-devel"
```
Comment (Vaelatern) [inline srcpkgs/libskk/template]: Yes, generally `-devel` packages are actually needed in `makedepends`.
`hostmakedepends` is for those packages where the host must run binaries during the build.
`makedepends` is for packages which get linked against, or header files included from, during the build.

**Comment by Gottox** [comment]
@Vaelatern any further comments?

@rail44 please make sure we can merge this PR.

**Comment by rail44** [comment]
rebased.

**Comment by Vaelatern** [comment]
No further comments.

**Comment by ebfe** [comment]
Needs a rebase.

**Comment by Vaelatern** [comment]
@ebfe If not for the rebase, would you consider accepting it?

**Comment by ebfe** [comment]
> @ebfe If not for the rebase, would you consider accepting it?
Sure, why not?

**Comment by Vaelatern** [comment]
Any objections to a `git-merge-pr` here?

**Comment by Gottox** [comment]
Good to go after a rebase :+1:

**Comment by rail44** [comment]
@Gottox 
Sorry for the delay in my noticed.
I've just rebased the branch :)

**Pull request merged**